package.json

```diff
@@ -11,9 +11,7 @@
     "showcase": "parcel showcase/public/index.html",
     "test": "elm-test",
     "i18n": "i18n-json-to-elm",
-    "percy": "percy exec -- cypress run",
-    "review": "npx elm-review src showcase review",
```

Comment:
if this is useful we could keep a script with
```
npx elm-review --template PaackEng/elm-review-config src

Reply:
Yeah, I didn't think about this, but I prefer to be able to run it through "npm run". Otherwise we'll have to teach everybody how to install it globally and that this command should be used to preview the same operation as the CI will do...